packages/radix-ui-themes/src/components/select.tsx

```diff
@@ -105,7 +105,6 @@ const SelectContent = React.forwardRef<SelectContentElement, SelectContentProps>
           <SelectPrimitive.Content
             data-accent-color={resolvedColor}
             sideOffset={4}
-            align="center"
```

Comment:
This is for when used with `position="popper"` you think start align looks better? I remember it looked a bit odd to me because of the tick offset, hence why I picked `center`

Reply:
Yeah I think left-aligned looks way more standard. There's a left alignment bias in the UIs generally, and center-aligned has felt super off to me whenever I noticed it.